crates/mipsy_interactive/src/interactive/commands/watchpoint.rs

```diff
@@ -1,7 +1,7 @@
 use crate::interactive::{error::CommandError, prompt};
 use std::{iter::successors, str::FromStr};
 
-use super::*;
+use super::{*, commands::handle_commands};
```

Comment:
is `commands::handle_commands` needed if there is already a glob import?

Reply:
you would think not but apparently it is. my guess is the glob import doesn't import recursively

Reply:
rust needs `use super::**;`